.github/workflows/performance-benchmarks-kurobako.yml

```diff
@@ -0,0 +1,144 @@
+name: Performance Benchmarks with kurobako
+
+on:
+  workflow_dispatch:
+    inputs:
+      sampler-list:
+        description: 'Sampler List: A list of samplers to check the performance. Should be a whitespace-separated list of Optuna samplers. Each sampler must exist under `optuna.samplers` or `optuna.integration`.'
+        required: false
+        default: 'RandomSampler TPESampler'
+      sampler-kwargs-list:
+        description: 'Sampler Arguments List: A list of sampler keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{} {\"multivariate\":true,\"constant_liar\":true}'
+      pruner-list:
+        description: 'Pruner List: A list of pruners to check the performance. Should be a whitespace-separated list of Optuna pruners. Each pruner must exist under `optuna.pruners`.'
+        required: false
+        default: 'NopPruner'
+      pruner-kwargs-list:
+        description: 'Pruner Arguments List: A list of pruner keyword arguments. Should be a whitespace-separated list of json format dictionaries.'
+        required: false
+        default: '{}'
+      extra-requirements:
+        description: 'Extra Requirements: A list of extra requirements to run the benchmark. Should be a whitespace-separated list of packaged which can be installed by `pip`.'
```

Comment:
Is this argument commonly necessary and if so when?

Reply:
I initially thought that if users implement a new sampler with additional libraries and want to take a bench, they need to specify those libraries. But, I've noticed that they should write the dependencies in setup.py, and then they don't have to specify those libraries as this argument. 

Thanks for the comment. I remove it.